resource-managers/kubernetes/core/pom.xml

```diff
@@ -52,6 +52,12 @@
       <type>test-jar</type>
     </dependency>
 
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>volcano-model-v1beta1</artifactId>
+      <version>${kubernetes-client.version}</version>
+    </dependency>
```

Comment:
Technically, Apache Spark uses three options when it embraces new features. I'd prefer (2). Shall we put this under `-Pvolcano` profile please?
1. `External module` like `Kafka Streaming`
2. `Profile` like `HiveThriftServer` or `SparkR`
3. Built-in module.

Reply:
Yep, `(2)` is also ok to me